src/helpers.ts

```diff
@@ -66,22 +66,27 @@ function printDoc(gridResult: [], columns: ColumnModel[], gridName: string): voi
     documentToPrint.document.close();
 }
 
-function exportFile(gridResult: [], columns: ColumnModel[]): void {
+function exportFile(gridResult: [], columns: ColumnModel[], gridName: string): void {
```

Comment:
Please make gridName nullable so that it doesn't produce any errors on existing implementations.